### ISSUE TITLE
PointCloudPoint Confidence value - what is it exactly

### ISSUE THREAD

**Issue by FUBAR2077** [google-ar/arcore-unity-sdk#480]
Hello, I wanted to ask few questions:

1. What exactly is confidence value?

2. If confidence value is a parameter telling us if certain PointCloudPoint is reliable then why it's not updating in runtime?

**Comment by fredsa** [comment]
For advice and general questions, please use the `ARCore` tag on
Stack Overflow: https://stackoverflow.com/questions/tagged/ARCore

**Comment by FUBAR2077** [comment]
Ok, let me rephrase - I think there is a bug because PointCloudPoint confidence value is not updating at runtime

**Comment by fredsa** [comment]
If the confidence values are not changing, please feel free to file a new issue. I've verified that the values due update correctly in the 1.6.0 SDK, both on device, and when running Instant Preview.
